plutus-core/untyped-plutus-core/src/UntypedPlutusCore.hs

```diff
@@ -15,7 +15,8 @@ import UntypedPlutusCore.Simplify as Export
 import UntypedPlutusCore.Size as Export
 import UntypedPlutusCore.Subst as Export
 
-import PlutusCore qualified as PLC
+import PlutusCore.Core qualified as PLC (defaultVersion)
```

Comment:
If it's imported qualified, then what's the point of restricting which items are brought into scope?